inst/extdata/variable_details.csv

```diff
@@ -561,6 +561,16 @@ COPD_Emph_der,COPD_Emph_der_cat3_1,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs
+DEN_132,DEN_132_cat7_3,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,3,7,2 to < 3 years,2 to < 3 years,N/A,3,2 to < 3 years,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_4,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,4,7,3 to < 4 years,3 to < 4 years,N/A,4,3 to < 4 years,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_5,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,5,7,4 to < 5 years,4 to < 5 years,N/A,5,4 to < 5 years,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_6,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,6,7,5 years or more,5 years or more,N/A,6,5 years or more,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_7,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,7,7,Never,Never,N/A,7,Never,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_NA::a,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,NA::a,7,not applicable,not applicable,N/A,96,not applicable,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_NA::b,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,NA::b,7,missing,missing,N/A,"[97,99]",don't know (97); refusal (98); not stated (99),Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_NA::b,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,NA::b,7,missing,missing,N/A,else,else,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,

```

Comment:
@caitlink12 Can you do the same as in [this comment](https://github.com/Big-Life-Lab/cchsflow/pull/140/commits/db122101b4616be897bba7504d1a33352759da97#r2291391462)